use.go

```diff
@@ -8,7 +8,9 @@ func (n *Notify) useService(service Notifier) {
 	n.notifiers = append(n.notifiers, service)
 }
 
-// UseService adds a given service to the notifiers services list.
-func (n *Notify) UseService(service Notifier) {
-	n.useService(service)
+// UseService adds the given service(s) to the notifiers services list.
+func (n *Notify) UseService(service ...Notifier) {
```

Comment:
Please change the method name from `UseService` to `UseServices` to match the naming style of the `Notifier.AddReceivers` method.

```suggestion
// UseServices adds the given service(s) to the notifiers services list.
func (n *Notify) UseServices(service ...Notifier) {
```